### ISSUE TITLE
[BUG] Missing javax.activation when run on JRE 11

### ISSUE THREAD

**Issue by sideeffect42** [eXist-db/exist#4126]
**Describe the bug**
When starting the current eXist-db release (5.3.1) on JRE 11 the following exception occurs at startup:

```
2021-12-15 11:02:57,790 [main] INFO  (Configuration.java [<init>]:184) - Reading configuration from file /opt/exist-5.3.1/etc/conf.xml 
2021-12-15 11:02:58,052 [main] INFO  (Configuration.java [configureStartup]:1260) - Registered StartupTrigger: org.exist.security.BouncyCastleJceProviderStartupTrigger 
2021-12-15 11:02:58,053 [main] INFO  (Configuration.java [configureStartup]:1260) - Registered StartupTrigger: org.exist.protocolhandler.URLStreamHandlerStartupTrigger 
2021-12-15 11:02:58,054 [main] INFO  (Configuration.java [configureStartup]:1260) - Registered StartupTrigger: org.exist.extensions.exquery.restxq.impl.RestXqStartupTrigger 
2021-12-15 11:02:58,056 [main] INFO  (Configuration.java [configureStartup]:1260) - Registered StartupTrigger: org.exist.repo.AutoDeploymentTrigger 
2021-12-15 11:02:58,269 [main] ERROR (Configuration.java [lookupModuleClass]:503) - Module http://exist-db.org/xquery/mail could not be initialized due to a missing dependancy (NoClassDefFoundError): javax/activation/MimeTypeParseException 
java.lang.NoClassDefFoundError: javax/activation/MimeTypeParseException
	at org.exist.xquery.modules.mail.MailModule.<clinit>(MailModule.java:72) ~[exist-mail-5.3.1.jar:5.3.1]
	at java.lang.Class.forName0(Native Method) ~[?:?]
	at java.lang.Class.forName(Class.java:315) ~[?:?]
	at org.exist.util.Configuration.lookupModuleClass(Configuration.java:489) [exist-core-5.3.1.jar:5.3.1]
	at org.exist.util.Configuration.loadModuleClasses(Configuration.java:455) [exist-core-5.3.1.jar:5.3.1]
	at org.exist.util.Configuration.configureXQuery(Configuration.java:391) [exist-core-5.3.1.jar:5.3.1]
	at org.exist.util.Configuration.<init>(Configuration.java:276) [exist-core-5.3.1.jar:5.3.1]
	at org.exist.util.SingleInstanceConfiguration.<init>(SingleInstanceConfiguration.java:56) [exist-core-5.3.1.jar:5.3.1]
	at org.exist.util.SingleInstanceConfiguration.<init>(SingleInstanceConfiguration.java:52) [exist-core-5.3.1.jar:5.3.1]
	at org.exist.jetty.JettyStart.run(JettyStart.java:207) [exist-core-5.3.1.jar:5.3.1]
	at org.exist.jetty.JettyStart.main(JettyStart.java:103) [exist-core-5.3.1.jar:5.3.1]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.exist.start.Main.invokeMain(Main.java:153) [exist-start-5.3.1.jar:5.3.1]
	at org.exist.start.Main.runEx(Main.java:292) [exist-start-5.3.1.jar:5.3.1]
	at org.exist.start.Main.run(Main.java:158) [exist-start-5.3.1.jar:5.3.1]
	at org.exist.start.Main.main(Main.java:95) [exist-start-5.3.1.jar:5.3.1]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
	at org.codehaus.mojo.appassembler.booter.AppassemblerBooter.executeMain(AppassemblerBooter.java:257) [appassembler-booter-2.1.0.jar:?]
	at org.codehaus.mojo.appassembler.booter.AppassemblerBooter.main(AppassemblerBooter.java:77) [appassembler-booter-2.1.0.jar:?]
Caused by: java.lang.ClassNotFoundException: javax.activation.MimeTypeParseException
	at java.net.URLClassLoader.findClass(URLClassLoader.java:476) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:589) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:522) ~[?:?]
	... 25 more
2021-12-15 11:02:58,302 [main] INFO  (Configuration.java [configureValidation]:1470) - Add catalog uri file:///opt/exist-5.3.1/etc/webapp//WEB-INF/catalog.xml 
2021-12-15 11:02:58,303 [main] INFO  (GrammarPool.java [<init>]:54) - Initializing GrammarPool. 
2021-12-15 11:02:58,304 [main] INFO  (JettyStart.java [run]:211) - Configuring eXist from /opt/exist-5.3.1/etc/conf.xml 
```

**Expected behavior**
No such exception occurs, i.e. the activation library is shipped with eXist for compatibility with newer Java versions.

**Context (please always complete the following information):**
 - OS: Devuan 4 (chimaera)
 - eXist-db version: 5.3.1
 - Java version: openjdk version "11.0.13" 2021-10-19

**Comment by dizzzz** [comment]
Please could you describe how you start eXist-db ? I am surprised as @adamretter addressed this subject some time ago

**Comment by sideeffect42** [comment]
@dizzzz Are you referring to #3969? This does indeed seem to be the same bug, but I hadn't noticed this because it was closed already.

I'm using the bundled `bin/startup.sh` to start eXist.

I believe this issue can be closed and I'll wait for the 5.4.0 release then.

**Comment by dizzzz** [comment]
I need to check but I hoped it would be in 531 already. But 531 is probably a log4j fix only, I need to check

**Comment by duncdrum** [comment]
i can reproduce this on my latest docker experiments (which use 5.3.1), starting with `java org.exist…` on deb11 aka distroless

the error only appears on J11, J8 seems fine, although we don't seem to have any tests for the module see #4156

**Comment by duncdrum** [comment]
fixed in 5.4.0 and up
